windows/win_file.ps1

```diff
@@ -50,6 +50,33 @@ $result = New-Object psobject @{
     changed = $FALSE
 }
 
+# Used to delete directories and files with logic on handling symbolic links
+Function Delete($file)
+{
+    $filepath = $file.FullName
+    If ( $file.Attributes.ToString().contains("ReparsePoint") )
```

Comment:
Since this is an enum, it'd be better to actually test the enum value instead of using magic strings, eg: `If ($file.Attributes -band [System.IO.FileAttributes]::ReparsePoint)`